test/udsource-e2e/udsource_test.go

```diff
@@ -39,6 +39,10 @@ func (s *UserDefinedSourceSuite) TestSimpleSourceJava() {
 	s.testSimpleSource("java")
 }
 
+func (s *UserDefinedSourceSuite) TestSimpleSourcePython() {
+	s.testSimpleSource("python")
+}
```

Comment:
Can we run them in parallel? something similar to https://github.com/numaproj/numaflow/pull/1179/files#diff-9adbbdcedf1c205830aacf0cee5f595e1cb980745f7b3715ddf6ffd04809c6f1R127